stock_inventory_preparation_filter/models/stock_inventory.py

```diff
@@ -46,6 +47,11 @@ def _selection_filter(self):
         res_filters.append(('empty', _('Empty list')))
         return res_filters
 
+    filter = fields.Selection(
```

Comment:
isn't this a reserved name ?

Reply:
Well, yes, but blame Odoo that puts that name to the field...

Reply:
Ok didn't realised it was an overridden one.

Reply:
I don't think we're shadowing the `filter` function since we're just defining an object property, don't you think?